### DISCUSSION TITLE
Copilot showing me completions in wrong language.

### DISCUSSION THREAD

**Discussion by devoidoflight** [community/community#18806] [Copilot Conversations]
Hey! I'm using Copilot on DataGrip. I'm working with SQL and I've never write java. But after the update copilot started showing me java completions. How can i change it back to SQL?

**Comment by wangshean** [comment]
Could you please paste some sample code for our debugging?

**Reply by rajeee** [reply]
For example if you type: ` current_file_path =` in Julia
CoPilot will suggest  `current_file_path = Path(__FILE__)` which I believe is coming from python.

**Comment by ice1000** [comment]
I often hear people saying that copilot suggests Haskell completions in OCaml files
